site/README.md

```diff
@@ -10,6 +10,13 @@ The service this website provides is as follows:
  1. We use the version number to redirect the traffic to the right update site,
     among all the ones that we generate.
 
+### What happens when 'Update Now' is pressed?
```

Comment:
Third level header directly below a first level header makes no sense.

Beyond formatting, this is also unrelated to anything else going on in this document.